EU4toV2/Source/EU4World/Leader/EU4Leader.h

```diff
@@ -10,17 +10,16 @@ namespace EU4
 	{
 	public:
 		Leader() = default;
-		Leader(std::istream& theStream);
+		explicit Leader(std::istream& theStream, const std::string& leaderClass);
```

Comment:
I'm not seeing where leaderClass is used for anything. Is that for the future? Something not quite complete? Something started and then dropped?

Reply:
It's monarch|queen|heir|leader|etc. It's used within the class to see if the leader we're extracting dynasty from is a monarch, otherwise skip. There is no syntax difference in eu4 saves between rulers and leaders.

Reply:
correction - it _used_ to be used inside to see if it's a monarch but then I divorced leaders themselves from actual dynasties to simplifiy the design and fix bugs. I've dumped this now.